src/DataValues/QuantityValue.php

```diff
@@ -283,13 +292,17 @@ public function getUncertainty() {
 	 *
 	 * @since 0.1
 	 *
-	 * @return DecimalValue
+	 * @return DecimalValue|null
 	 */
 	public function getUncertaintyMargin() {
+		if ( $this->upperBound === null ) {
```

Comment:
Or perhaps we could move bounds to a subclass, BoundedQuantity, instead of checking for bounds in every method.